backend/stakepoold/server.go

```diff
@@ -361,6 +361,10 @@ func runMain(ctx context.Context) error {
 		}()
 	}
 
+	if cfg.Prometheus {
+		spd.ExportPrometheusMetrics(ctx, wg, cfg.PrometheusWait, cfg.PrometheusListen)
```

Comment:
can you log an info here, something like "Serving Prometheus metrics at http://{ip}:{port}/metrics"